lib/ansible/plugins/callback/foreman.py

```diff
@@ -107,12 +107,13 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
         else:
             self._disable_plugin('The `requests` python module is not installed.')
 
-        if self.FOREMAN_URL.startswith('https://'):
-            if not os.path.exists(self.FOREMAN_SSL_CERT[0]):
-                self._disable_plugin('FOREMAN_SSL_CERT %s not found.' % self.FOREMAN_SSL_CERT[0])
+        if self.ssl_verify:
```

Comment:
There's no need for additional indenting:
```suggestion
        if self.ssl_verify and self.FOREMAN_URL.startswith('https://'):
```